npm/vue/src/index.ts

```diff
@@ -72,24 +72,12 @@ type MountingOptions<Props, Data = {}> = Omit<VTUMountingOptions<Props, Data>, '
 
 export type CyMountOptions<Props, Data = {}> = MountingOptions<Props, Data>
 
-Cypress.on('run:start', () => {
-  // `mount` is designed to work with component testing only.
-  // it assumes ROOT_SELECTOR exists, which is not the case in e2e.
-  // if the user registers a custom command that imports `cypress/vue`,
-  // this event will be registered and cause an error when the user
-  // launches e2e (since it's common to use Cypress for both CT and E2E.
-  // https://github.com/cypress-io/cypress/issues/17438
-  if (Cypress.testingType !== 'component') {
-    return
-  }
-
-  Cypress.on('test:before:run', () => {
-    Cypress.vueWrapper?.unmount()
-    const el = getContainerEl()
+const cleanup = () => {
+  Cypress.vueWrapper?.unmount()
```

Comment:
If there's no `Cypress.vueWrapper` that would indicate it's the first mount, so possibly we just return here instead of touching the DOM?

Actually wondered, as part of cleanup do we want to unset `Cypress.vueWrapper` and `Cypress.vue`? Probably pointless since mount will overwrite them & cleanup is always called before mount.

Reply:
I'm going to remove the `innerHtml = ''`, more consistent with other frameworks. I'll add the cleanup as well.